### ISSUE TITLE
can't make the alises to work (using Declarative Approach)

### ISSUE THREAD

**Issue by rabin-io** [fabriziopandini/vagrant-compose#15]
```yaml
# cluster-setup.yaml
kubernetes:
  box: centos/7
  master:
    aliases: 
      - x
      - y
    instances: 1
  minions:
    instances: 0
```
```ruby
# Vagrantfile
Vagrant.configure(2) do |config|
    #load cluster definition
    config.cluster.from("cluster-setup.yaml")

   config.hostmanager.enabled = false          
   config.hostmanager.manage_host = true       
   config.hostmanager.include_offline = true  

    config.cluster.nodes.each do |node|
      config.vm.define "#{node.boxname}" do |node_vm|
        node_vm.vm.box = "#{node.box}"
        node_vm.vm.network :private_network, ip: "#{node.ip}"
        node_vm.vm.hostname = "#{node.fqdn}"
        node_vm.hostmanager.aliases = node.aliases
        node_vm.vm.provision :hostmanager

        node_vm.vm.provider "virtualbox" do |vb|
          vb.name = "#{node.boxname}"  
          vb.memory = node.memory
          vb.cpus = node.cpus
        end            
      end
    end
  config.cluster.debug
end
```

```shell
$ vagrant up
==> cluster kubernetes with 1 nodes
        master1 accessible as master1.vagrant master1 172.31.0.101 => [centos/7, 1 cpus, 256 memory]
...
==> master1: Preparing network interfaces based on configuration...
    master1: Adapter 1: nat
    master1: Adapter 2: hostonly
==> master1: Forwarding ports...
    master1: 22 (guest) => 2200 (host) (adapter 1)
...
==> master1: Setting hostname...
==> master1: Configuring and enabling network interfaces...
==> master1: Rsyncing folder: /tmp/x/ => /vagrant
==> master1: Running provisioner: hostmanager...

$ vagrant ssh
[vagrant@master1 ~]$ cat /etc/hosts
127.0.0.1	master1.vagrant	master1
127.0.0.1   localhost localhost.localdomain localhost4 localhost4.localdomain4
::1         localhost localhost.localdomain localhost6 localhost6.localdomain6

## vagrant-hostmanager-start
172.31.0.101	master1.vagrant
172.31.0.101	master1
## vagrant-hostmanager-end
```

was expecting x & y as aliases in the `hosts` file.

**Comment by fabriziopandini** [comment]
Thanks for this issue. The fix is include in v0.7.5 just published.
To upgrade run `vagrant plugin update vagrant-compose`
Let me know if there are still problems

**Comment by rabin-io** [comment]
thanks, working.
